### ISSUE TITLE
Lantern footsteps appear to play only for other players, not self

### ISSUE THREAD

**Issue by LessThanBenji** [Sollace/Presence-Footsteps#225]
When walking over or along lanterns, footsteps sounds are not played. However, the footsteps of other player's walking over or along lanterns is played and audible.

This is on 1.8.2, in Fabric 0.14.21 for 1.19.4.

**Comment by Sollace** [comment]
Not really sure how this is possible since it's the same method that handles all players, regardless of who it is. Have you tried moving around more? Steps play based on distance travelled, so if you're just inching around on one block you probably won't hear them.

**Comment by LessThanBenji** [comment]
Yep, tried all sorts of movements. This strange bug affects both me and other players (ie. we can all only hear other players' footsteps on lanterns, never our own). All of us are using the same version (1.8.2) in the same modded environment (0.14.21 on 1.19.4). As far as we can tell only lanterns are affected.

**Comment by Sollace** [comment]
Lanterns also have very small hitboxes. Hm...

**Comment by Sollace** [comment]
Cannot reproduce in 1.20.1
